Geometry/GEMGeometry/src/GEMSuperChamber.cc

```diff
@@ -37,3 +37,26 @@ const GEMChamber* GEMSuperChamber::chamber(int isl) const {
   }
   return nullptr;
 }
+
+float GEMSuperChamber::computeDeltaPhi(const LocalPoint& position, const LocalVector& direction) const {
+  auto extrap = [](const LocalPoint& point, const LocalVector& dir, double extZ) -> LocalPoint {
+    double extX = point.x() + extZ * dir.x() / dir.z();
```

Comment:
Please prevent division by zero by checking `dir.z()`.

Reply:
@watson-ij Please address this issue.

Reply:
If you look at the latest version of the code, there's a check for z==0 above this line

Reply:
Sorry, must be going blind. ;-).  Looks good.